radiography/src/main/java/radiography/SkipIdsViewFilter.kt

```diff
@@ -0,0 +1,15 @@
+package radiography
+
+import android.view.View
+import java.util.Arrays
+
+/**
+ * Filters out views with ids matching [skippedIds] from the output of [Radiography.scan].
+ */
+class SkipIdsViewFilter(private vararg val skippedIds: Int) : ViewFilter {
+
+  override fun matches(view: View): Boolean {
+    val viewId = view.id
+    return (viewId == View.NO_ID || Arrays.binarySearch(skippedIds, viewId) < 0)
```

Comment:
Nit: I think `binarySearch` is an extension on arrays in Kotlin (my bad for not doing this in the kotlin conversion PR)